src/Cookie/SameSite.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Spiral\Cookies\Cookie;
+
+class SameSite
```

Comment:
What do you think about add `isSecure(): bool` method?

Reply:
It's not used, I need it just to define the correct `sameSite` value because `sameSite` depends on `isSecure`